app/src/main/java/campus/tech/kakao/map/MapRepository.kt

```diff
@@ -0,0 +1,38 @@
+package campus.tech.kakao.map
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+
+class MapRepository(context: Context) {
+    private var localDB: PlacesDBHelper
+
+    init {
+        localDB = PlacesDBHelper(context)
+        val dbFile = context.getDatabasePath("PlaceTable.db")
+        if (!dbFile.exists())
+            insertInitialData()
+    }
+
+    private fun insertInitialData() {
+        for (i in 1..30) {
+            val cafe = Place("카페" + i, "서울 성동구 성수동 " + i, "카페")
+            val drugStore = Place("약국" + i, "서울 강남구 대치동 " + i, "약국")
```

Comment:
문자열을 동적으로 구성할 땐 아래와 같이 String Template을 이용해 좀 더 코틀린스럽게 작성하시는 것을 추천드려요.
```suggestion
            val drugStore = Place("약국$i", "서울 강남구 대치동 $i", "약국")
```
🔗 코틀린 > [String templates](https://kotlinlang.org/docs/strings.html#string-templates)